@remilapeyre, the builds are failing because of the new documentation you added to the `.rst` file for `unwrap`.  It is seeing it as 'suspicious' markup.  Here is the output of the failure:

```
Warning, treated as error:
[library/urllib.parse:379] ":scheme" found in "<URL:scheme://host/path>"
Makefile:44: recipe for target 'build' failed
make[1]: *** [build] Error 2
make[1]: Leaving directory '/home/travis/build/python/cpython/Doc'
Suspicious check complete; look for any errors in the above output or in build/suspicious/suspicious.csv.  If all issues are false positives, append that file to tools/susp-ignored.csv.
Makefile:105: recipe for target 'suspicious' failed
```

So, based on the suggestion there, you need to edit `susp-ignored.csv` in `https://github.com/python/cpython/tree/master/Doc/tools`.  Here's a [link](https://github.com/python/cpython/blob/master/Doc/tools/extensions/suspicious.py) to the checks, for more info.

--- a/Doc/tools/susp-ignored.csv
+++ b/Doc/tools/susp-ignored.csv
@@ -235,6 +235,8 @@ library/urllib.request,,:close,Connection:close
 library/urllib.request,,:port,:port
 library/urllib.request,,:lang,"xmlns=""http://www.w3.org/1999/xhtml"" xml:lang=""en"" lang=""en"">\n\n<head>\n"
 library/urllib.request,,:password,"""joe:password@python.org"""
+library/urllib.parse,,:scheme,<URL:scheme://host/path>
+library/urllib.parse,,:scheme,URL:scheme://host/path
 library/uuid,,:uuid,urn:uuid:12345678-1234-5678-1234-567812345678
 library/venv,,:param,":param nodist: If True, setuptools and pip are not installed into the"
 library/venv,,:param,":param progress: If setuptools or pip are installed, the progress of the"

